libsrc/allegro/include/allegro/file.h

```diff
@@ -70,9 +70,9 @@ struct PACKFILE_VTABLE
    AL_METHOD(int, pf_fclose, (void *userdata));
    AL_METHOD(int, pf_getc, (void *userdata));
    AL_METHOD(int, pf_ungetc, (int c, void *userdata));
-   AL_METHOD(long, pf_fread, (void *p, long n, void *userdata));
+   AL_METHOD(int, pf_fread, (void *p, int n, void *userdata));
    AL_METHOD(int, pf_putc, (int c, void *userdata));
-   AL_METHOD(long, pf_fwrite, (AL_CONST void *p, long n, void *userdata));
+   AL_METHOD(int, pf_fwrite, (AL_CONST void *p, int n, void *userdata));
```

Comment:
It's better to make these size_t, for compliance with classic stream reading and writing.

Reply:
Switched to size_t!